src/posts/how-to-use-JAWS-on-Windows

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
One issue we'll need to solve is the file name. This needs an `.md` file extension so that it can compiled and previewed. While we're at it, let's make the filename all lowercase.